src/cinder/Triangulate.cpp

```diff
@@ -66,15 +73,16 @@ Triangulator::Triangulator()
 void Triangulator::allocate()
 {
 	mAllocated = 0;
-	
-	TESSalloc ma;
-	memset( &ma, 0, sizeof(ma) );
-	ma.memalloc = stdAlloc;
-	ma.memfree = stdFree;
-	ma.userData = (void*)&mAllocated;
-	ma.extraVertices = 2560; // realloc not provided, allow 256 extra vertices.
-
-	mTess = shared_ptr<TESStesselator>( tessNewTess( &ma ), tessDeleteTess );
+
+	mAlloc = shared_ptr<TESSalloc>( new TESSalloc() );
```

Comment:
If you really want to use a shared_ptr instead of a unique_ptr may I suggest using `std::make_shared`?